packages/x-components/static-docs/build-search-ui/web-archetype-integration-guide.md

```diff
@@ -131,7 +131,7 @@ attribute `src` so that it points to the staging environment as follows:
 <script>
   window.initX = {
     instance: 'my-store',
-    env: 'live', // Note that here you are using a production API with the staging version of Interface X
```

Comment:
I would keep this message either here or in another place. It is a point that I found users struggle. They don't realise that they can load the staging x-components with the production API or viceversa.